src/components/notification/notificationCard/index.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+
+// Components
+import { NotificationCardContent } from './NotificationCardContent';
+
+// MUI
+import { Card, CardActionArea } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import style from '../../../style';
+
+// API
+import { markNotificationRead } from '../../../util/fetch/user';
+
+/** Displays icon appropriate to notification type
+ *
+ * @type {React.FunctionComponent}
+ * @param {object} props
+ * @param {object} props.classes
+ * @param {UserNotification} props.notification
+ * @param {Post} props.post
+ * @param {UserComment} props.comment
+ * @param {Reac} history
+ */
+export const NotificationCardView = ({
+  classes = {},
+  notification = {},
+  post = {},
+  comment = {},
+  history,
```

Comment:
`const history = useHistory()`

no reason to pass this guy

Reply:
Fixed